os/tools/mkbinheader.py

```diff
@@ -173,6 +187,10 @@ def get_static_ram_size(bin_type):
     print("Dynamic ram size : %d, Main stack size : %d" %(int(dynamic_ram_size), int(main_stack_size)))
     sys.exit(1)
 
+config_path = os.getenv('TOPDIR') + '/.config'
+BM_PRIORITY_MAX = get_config_value(config_path, "CONFIG_BM_PRIORITY_MAX=")
```

Comment:
How about setting the default BM_PRIORITY_MAX and MIN if `get_config_value` returns fail?